File: PurchaseSale/Api/PurchaseSale.Entity/PurchaseSale.cs
using OpenDataAccess.Entity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PurchaseSale.Entity
{
    [TableProperty(Name = "v_PurchaseSale", PrimaryKey = "SaleDate")]
    [RequestMethod(IsDelete = false, IsPost = false, IsPut = false)]
    public class PurchaseSale : EntityModel, IEntity
    {
        public DateTime SaleDate { get; set; }
        public int SaleYear { get; set; }
        public string SaleMonth { get; set; }
        public string SaleDay { get; set; }
        public decimal SaleAmount { get; set; }
        public decimal SaleBidAmount { get; set; }
        public decimal SaleDueAmount { get; set; }
        public decimal SaleProfit { get; set; }
        public decimal SaleRealAmount { get; set; }
        public decimal SaleShouldAmount { get; set; }
        public decimal PurchaseAmount { get; set; }
        public decimal PurchaseDueAmount { get; set; }
        public decimal PurchaseRealAmount { get; set; }
        public decimal PurchaseShouldAmount { get; set; }
        public decimal ShouldBalance { get; set; }
        public decimal RealBalance { get; set; }
    }
}

File: WebHook/configs/entities/PurchaseSale.js
module.exports = {
    Name: "PurchaseSale",
    PrimaryKey: "SaleDate",
    Properties: GetProperties()
}

function GetProperties() {
    return [
        GetProperty("SaleDate", "日期"),
        GetProperty("SaleYear", "年"),
        GetProperty("SaleMonth", "月"),
        GetProperty("SaleDay", "天"),
        GetProperty("SaleAmount", "销售商品金额"),
        GetProperty("SaleBidAmount", "销售商品成本"),
        GetProperty("SaleDueAmount", "销售待收金额"),
        GetProperty("SaleProfit", "销售商品利润"),
        GetProperty("SaleRealAmount", "销售实收金额"),
        GetProperty("SaleShouldAmount", "销售应收金额"),
        GetProperty("PurchaseAmount", "采购商品金额"),
        GetProperty("PurchaseDueAmount", "采购待付金额"),
        GetProperty("PurchaseRealAmount", "采购实付金额"),
        GetProperty("PurchaseShouldAmount", "采购应付金额"),
        GetProperty("ShouldBalance", "应结余金额"),
        GetProperty("RealBalance", "实结余金额")
    ]
}

function GetProperty(Name, Label) { return { Name, Label } }